js/renovation/ui/toolbar/toolbar.tsx

```diff
@@ -26,4 +24,27 @@ export const viewFunction = ({
   view: viewFunction,
 })
 
-export class Toolbar extends JSXComponent<ToolbarProps>() {}
+export class Toolbar extends JSXComponent<ToolbarProps>() {
+  @Consumer(ConfigContext)
+  config?: ConfigContextValue;
+
+  get fixedProps(): ToolbarProps {
```

Comment:
What does 'fixed' word mean here?
Why it is easier to understand what does it means than with the 'props1' name?

Reply:
Yep, current name is difficult to understand. Renamed it to componentProps